### PR TITLE
Partner Updates - Pt. 11

### PR DESCRIPTION
**Pull request by KellieHawks**
Updates to Neura actions, neura insights, digioh, kubit, iterate, treasure data, zeotap, census, rudderstack

### CODE REVIEW

File: _docs/_partners/data_and_infrastructure_agility/customer_data_platform/rudderstack.md [hunk 1]
```diff
@@ -11,142 +11,94 @@ search_tag: Partner
...
-{: .reset-td-br-1 .reset-td-br-2}
+To start sending data to Braze, you first need to make sure a source is set up in your RudderStack app. Visit [RudderStack](https://rudderstack.com/docs/connections/adding-source-and-destination-rudderstack/) to learn how to set up your data source.
 
-{% alert note %} 
-Learn more about RudderStack's connection modes and the benefits of each [here](https://rudderstack.com/docs/connections/rudderstack-connection-modes/).
-{% endalert %}
+### Step 2: Configure destination
 
-### Step 2.1a: Side-by-side integration (device mode)
+Now that you have your data source set up, in the RudderStack dashboard, select **ADD DESTINATION** under **Destinations**. From the list of available destinations, select **Braze**and click **Next**.
```
Comment (bre-fitzgerald) [inline _docs/_partners/data_and_infrastructure_agility/customer_data_platform/rudderstack.md:37]:
```suggestion
Now that you have your data source set up, in the RudderStack dashboard, select **ADD DESTINATION** under **Destinations**. From the list of available destinations, select **Braze** and click **Next**.

```

File: _docs/_partners/data_and_infrastructure_agility/customer_data_platform/rudderstack.md [hunk 2]
```diff
@@ -11,142 +11,94 @@ search_tag: Partner
...
+You can choose to integrate RudderStack's web and native client-side libraries with Braze using either a side-by-side (device mode) integration or a server-to-server (cloud mode) integration.
 
-To enable the integration, set up your App Group's REST API Key and Braze's REST API endpoint in your Connection Settings (Refer to Step 2.2) on the RudderStack dashboard. You will also need to map the RudderStack methods to Braze (Refer to Step 3).
+- Integration type
+  - [Side-by-Side / Device Mode](#device-mode): RudderStack will send the event data to Braze directly from your client (browser or mobile application).
+  - [Server-to-Server / Cloud Mode](#cloud-mode): The Braze SDK sends the event data directly to RudderStack, which is then transformed and routed to Braze.
 
 {% alert note %} 
-All of RudderStack's server-side SDKs (Java, Python, Node.js, Go, Ruby) support only Cloud Mode. This is because their server-side SDKs operate in the RudderStack backend, and cannot load any Braze-specific SDK. 
+Learn more about RudderStack's connection modes and the benefits of [each](https://rudderstack.com/docs/connections/rudderstack-connection-modes/).
```
Comment (bre-fitzgerald) [inline _docs/_partners/data_and_infrastructure_agility/customer_data_platform/rudderstack.md:52]: I'd suggest linking "connection modes" or "RudderStack's connection modes" instead to parallel the article content/title.

```suggestion
Learn more about RudderStack's [connection modes](https://rudderstack.com/docs/connections/rudderstack-connection-modes/) and the benefits of each.
```

File: _docs/_partners/data_and_infrastructure_agility/customer_data_platform/treasure_data.md [hunk 3]
```diff
@@ -11,76 +11,77 @@ search_tag: Partner
...
-## Requirements
+## Prerequisites
 
-| Requirement | Origin | Access | Description |
-|---|---|---|---|
-| Braze API Key | Braze | You will need to create a new API Key.<br><br>This can be created in the __Developer Console -> API Settings -> Create New API Key__ with __users.track__, __users.delete__, __users.alias.new__, __users.identify__ permissions. | These API keys support the current feature to synchronize Treasure Data profiles to Braze, including: mapping external IDs, Upload Tracking, and opt-out  |
-| Braze REST Endpoint | Braze | [Braze REST Endpoint List][1] | Your REST Endpoint URL. Your endpoint will depend on the Braze URL for your instance. |
-| Treasure Data Account and Account Information | Treasure Data | [https://www.treasuredata.com/custom-demo/](https://www.treasuredata.com/custom-demo/) | You must have an active Treasure Data account to utilize their services with Braze |
-{: .reset-td-br-1 .reset-td-br-2 .reset-td-br-3  .reset-td-br-4}
+| Requirement | Description |
```
Comment (bre-fitzgerald) [inline _docs/_partners/data_and_infrastructure_agility/customer_data_platform/treasure_data.md:25]: Quick callout, just some minor inconsistencies in capitalization in this table. Specifically, you have both "REST API key" and "REST API Key", as well as "REST endpoint" and "REST Endpoint". I'd suggest the lowercase vs for both 😄

File: _docs/_partners/data_and_infrastructure_agility/customer_data_platform/zeotap.md [hunk 4]
```diff
@@ -11,38 +11,37 @@ search_tag: Partner
...
 
 ## Prerequisites
 
-| Requirement | Origin | Access | Description |
-|---|---|---|---|
-| Braze API Key | Braze | You will need to create a new API Key.<br><br>This can be created in the __Developer Console -> API Settings -> Create New API Key__ with __users.track__ permissions. | The Braze API Key will be used in the API calls to the Braze REST Endpoint URLs to authenticate the service. |
-| Braze REST Endpoint | Braze | [Braze REST Endpoint List][1] | Your REST Endpoint URL. Your endpoint will depend on the Braze URL for your instance. |
-|---|---|---|---|
-| Zeotap Account and Account Information | Zeotap | [Zeotap](https://zeotap.com/) | You must have an active Zeotap account to utilize their services with Braze. |
+| Requirement | Description |
```
Comment (bre-fitzgerald) [inline _docs/_partners/data_and_infrastructure_agility/customer_data_platform/zeotap.md:18]: Same comment as above on capitalization in this table :)

File: _docs/_partners/message_orchestration/channel_extensions/surveys/iterate.md [hunk 5]
```diff
@@ -12,43 +12,56 @@ search_tag: Partner
 
 > [Iterate](https://iteratehq.com) makes it easy to learn from your customers, offering smart, user-friendly research tools that look and feel like your brand.
 
-Iterate integrates with Braze by recording survey responses as custom user attributes in Braze allowing you to create powerful new audiences and segments.
+The Braze and Iterate integration lets you include iterate survey links in your email, push, or in-app messages. These links, once received, can automatically record and attribute Iterate survey responses as Braze custom user attributes, allowing you to create powerful new audiences and segments to use in your campaigns.
```
Comment (bre-fitzgerald) [inline _docs/_partners/message_orchestration/channel_extensions/surveys/iterate.md:15]:
```suggestion
The Braze and Iterate integration lets you include Iterate survey links in your email, push, or in-app messages. These links, once received, can automatically record and attribute Iterate survey responses as Braze custom user attributes, allowing you to create powerful new audiences and segments to use in your campaigns. 
```

File: _docs/_partners/message_orchestration/channel_extensions/surveys/iterate.md [hunk 6]
```diff
@@ -12,43 +12,56 @@ search_tag: Partner
...
+## Prerequisites
 
-To connect Braze with Iterate you'll need the following three items:
-
-| Requirement | Origin | Description |
-|---|---|---|
-| Braze API Key | Braze | You will need to create a new API Key.<br><br>This can be created in the __Developer Console -> API Settings -> Create New API Key__ with __users.track__ permissions.
-| [Braze REST Endpoint][6] | Braze | Your REST Endpoint URL. Your endpoint will depend on the Braze URL for your instance. |
-| Iterate Account | Iterate | You will need to have a Iterate account. Visit their [website](https://iteratehq.com/) to get started. |
+| Requirement | Origin |
```
Comment (bre-fitzgerald) [inline _docs/_partners/message_orchestration/channel_extensions/surveys/iterate.md:19]: Same comment as above on capitalization in this table :)

**Review by bre-fitzgerald — CHANGES_REQUESTED** [review]
Looks great! 🙌  Left a few suggestions/comments for your review :)

File: _docs/_partners/data_and_infrastructure_agility/customer_data_platform/census.md [hunk 7]
```diff
@@ -11,49 +11,61 @@ search_tag: Partner
...
 
 | Requirement | Description |
-|---|---|
-| Braze API Key | You will need to create a new API Key.<br><br>This can be created in the **Developer Console -> API Settings -> Create New API Key** with **users.track** permissions. |
-| Braze REST Endpoint | Your REST [Endpoint URL][2]. Your endpoint will depend on the Braze URL for your instance. |
-| Census Account | An active Census account is required to take advantage of this Braze integration. |
-{: .reset-td-br-1 .reset-td-br-2}
+| --- | --- |
+| Census account | A [Census account][1] is required to take advantage of this partnership. |
+| Braze REST API key | A Braze REST API Key with all user data permissions(except for `users.delete`) and `segments.list` permissions. The permissions set may change as Census adds support for more Braze objects, so you may either want to grant more permissions now or plan to update these permissions in the future. <br><br> This can be created within the **Braze Dashboard -> Developer Console -> REST API Key -> Create New API Key** |
```
[the remark this answers is not in the document]
Comment (KellieHawks) [reply _docs/_partners/data_and_infrastructure_agility/customer_data_platform/census.md:23]: Nope! the `segments.list` endpoint is under its own segments permissions.

**Review by bre-fitzgerald — APPROVED** [review]
Lgtm!!

**Pull request merged**